doc/16-upgrading-icinga-2.md

```diff
@@ -8,6 +8,40 @@ Specific version upgrades are described below. Please note that version
 updates are incremental. An upgrade from v2.6 to v2.8 requires to
 follow the instructions for v2.7 too.
 
+## Upgrading to v2.14 <a id="upgrading-to-2-14"></a>
+
+### Dependencies and Redundancy Groups <a id="upgrading-to-2-14-dependencies"></a>
+
+Before Icinga v2.12 all dependecies were cumulative.
+I.e. the parent was considered reachable only if no dependency was violated.
+In v2.12 and v2.13, all dependecies were redundant.
+I.e. the parent was considered unreachable only if no dependency was fulfilled.
+v2.14 restores the pre-v2.12 behavior and allows to override it.
+
+To stick to the behavior of v2.12 and v2.13, you could put this configuration in a global zone:
+
+```
+template Dependency default {
+    redundancy_group = "make all dependecies redundant"
+}
```

Comment:
I'd state that there's nothing magical about the value and that this just puts every dependency by default into the same group (and then maybe use a less exotic name like `default`).

Reply:
I'm afraid it’s not the default. OK, the pre-2.14 default, but not the current default.

Reply:
It's not the default in the sense that Icinga 2 will assume this if you don't set it but if you add such a default template, it will be the default for all your dependencies unless you set something else.

I would rather prefer some more explanation around the config snippet instead of squeezing the explanation into a string value inside.

Reply:
1. What's the difference?

Reply:
> 1. What's the difference?

If that's obvious, I don't think I understood that question correctly. Please rephrase it/make it more verbose.

Reply:
> It's not the default in the sense that Icinga 2 will assume this if you don't set it but if you add such a default template, it will be the default for all your dependencies unless you set something else.

IMAO this is obvious. Let's forget all confusion since https://github.com/Icinga/icinga2/pull/9760#discussion_r1219770770 and look at the current state of this. (The above three-liner is outdated.) Is there anything non-obvious in the config in the context of the text above it?

Reply:
[That comment](https://github.com/Icinga/icinga2/pull/9760#discussion_r1219770770) was my reaction on the change you did on this code snippet. I don't like how you're trying to address this comment by trying to squeeze as much information as possible into a 3 line config snippet instead of adding a few lines that explain what's going on.

Reply:
I have an even better idea not to repeat too much:

* #9784